### PR TITLE
Updated Readme.rst

### PR DESCRIPTION
**Pull request by sidntrivedi**

### CODE REVIEW

**Comment by 22shubh22** [comment]
Same problem as in your previous [commit.](https://github.com/khanchi97/Smvdu-Algos/pull/160). Image not displayed.

**Comment by sidntrivedi** [comment]
Sir, kindly see my code. I think I have written it correct. Not able to
find out why it is not being displayed.

File: README.rst [hunk 1]
```diff
@@ -45,5 +45,18 @@ an issue or if you need any help then feel free to ping us in gitter channel.And
...
  :align: middle
+ 
+ **Complexities of various Search & Sort Algos:**
+ 
+ *To get a better understanding of the best,average and worst case complexities of various search and sort algos,take a look at              the following images And to understand what is BIG O Complexity, read the following blog:
+ The Beginner's guide to Big O Complexity- https://rob-bell.net/2009/06/a-beginners-guide-to-big-o-notation/
+ 
+**Happy Coding.****GodSpeed..!!**
+ 
+=====
```
Comment (khanchi97) [inline README.rst:56]: Add a blank line after line no. 56 and 59 😉

Comment (sidntrivedi) [reply README.rst:56]: Okayzz Sir..

**Review by khanchi97 — APPROVED** [review]

**Pull request merged**